internal/versions/api_test.go

```diff
@@ -167,10 +167,6 @@ var _ = Describe("ListSupportedOpenshiftVersions", func() {
 			Version:          &common.TestDefaultConfig.ReleaseVersion,
 		}
 
-		// Production release version
-		releaseImage.Version = swag.String("4.8.12")
```

Comment:
I think the test needs some stable version (be it 4.9.0 or 4.12.1 or something else)